### ISSUE TITLE
Wait for user before starting ssh tunnel

### ISSUE THREAD

**Issue by mnlevy1981** [NCAR/pangeo#7]
It would be nice to have a `Press any key to continue...` prompt before attempting to open the ssh tunnel -- I often run this script and then do something else for the 30 seconds or minute that it takes to connect. By the time I come back to my terminal window, the ssh request to open the tunnel has timed out so I enter my yubikey token at

```
Running create_tunnel...
ssh -N -L localhost:8888:r6i6n33:8890 mlevy@cheyenne.ucar.edu
Tunnel PID: 66102
Use ^C to kill ssh tunnel
Waiting for tunnel to open
appending output to nohup.out
Token_Response:
```

and get

```
PID 66102 died!
```
